.eslintrc.js

```diff
@@ -13,5 +13,7 @@ module.exports = {
     es6: true
   },
   rules: {
+    'operator-linebreak': ['error', 'before', { 'overrides': { '=': 'ignore'} } ],
```

Comment:
I absolutely cannot stand this eslint rule. Will potentially fight to the death over this.